lib/linguist/generated.rb

```diff
@@ -301,7 +301,11 @@ def generated_go?
       return false unless extname == '.go'
       return false unless lines.count > 1
 
-      return lines[0].include?("Code generated by")
+      # old check for the transitional period (remove it later)
+      return true if lines[0].include?("Code generated by")
```

Comment:
Why do we need to keep this? The second rule is more general.

Reply:
The second rule would fail for lines where first would match (the first rule ignores `DO NOT EDIT.`). Didn't want to break the old behaviour. What are your thoughts on this? Should we just follow 2nd rule?

Reply:
It looks like the original intent was to match on go-bindata generated files (cf. #2426), which won't match the second rule. So let's keep the first rule!